lib/live_admin.ex

```diff
@@ -78,13 +78,31 @@ defmodule LiveAdmin do
   def record_label(record, resource) do
     case get_config(resource, :label_with, :id) do
       {m, f, a} -> apply(m, f, [record | a])
-      label when is_atom(label) -> Map.fetch!(record, label)
+      label when is_atom(label) -> Map.get(record, label) || Map.get(record, :id)
```

Comment:
also changed this so when `label_with` is set globally it doesn't crash on tables that don't have that field and instead defaults to :id